docs/model_embeddings.md

```diff
@@ -44,14 +44,24 @@ Step by step instructions to create embeddings for a single MGRS tile location
                              --trainer.precision=bf16-mixed \
                              --data.data_dir=s3://clay-tiles-02/02/27WXN \
                              --data.batch_size=32 \
-                             --data.num_workers=16
+                             --data.num_workers=16 \
+                             --output-patch-embeddings=False \
+                             --shuffle=False
```

Comment:
Does this work? I thought we have to pass `--model.init_args.shuffle=False` (untested).

```suggestion
                             --model.init_args.output_patch_embeddings=False \
                             --model.init_args.=False
```

Reply:
Looks like the correct argument is `model.output_patch_embeddings`

```
>>> python trainer.py fit -h | grep embeddings
...                                [--model.output_patch_embeddings OUTPUT_PATCH_EMBEDDINGS]
...  --model.output_patch_embeddings OUTPUT_PATCH_EMBEDDINGS
```